lib/trento/application/integration/discovery/payloads/cluster/sbd_discovery_payload.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Trento.Integration.Discovery.ClusterDiscoveryPayload.Sbd do
+  @moduledoc """
+  SBD field payload
+  """
+  alias Trento.Support.ListHelper
+
+  @required_fields []
+  use Trento.Type
+
+  deftype do
+    embeds_one :config, SBDConfig do
```

Comment:
I tend to agree with Xabi.

What I can say is that if we mapped ALL the fields and we're sure about that, we could also keep it like this.
Otherwise, if we are not fully confident we could be a bit less rigid as xabi is suggesting.

Reply:
Ok, I just checked a bit and there is indeed at least one more field that we are not tracking right now. Let's just switch it to a map.